lib/api_remote.ml

```diff
@@ -117,6 +117,23 @@ module Slack : Api.Slack = struct
     (* must read whole response to update lexer state *)
     ignore (Slack_j.read_ok_res s l)
 
+  (** [lookup_user ctx email] queries slack for a user profile with [email] *)
+  let lookup_user ~(ctx: Context.t) ~email =
+    log#info "looking up %s\n" email;
+    let build_error e = fmt_error "%s\nfailed to lookup Slack user" e in
```

Comment:
we prefer to keep log errors in one line to grep easier (I think it's been mentioned before)